src/rpcclient/rpcclient/darwin/fs.py

```diff
@@ -15,6 +16,22 @@ def do_stat(client, stat_name, filename: str):
         return stat64.parse_stream(buf)
 
 
+class DarwinRemotePath(RemotePath):
+    _flavour = type(Path())._flavour
```

Comment:
if this is taken from a credible internet source, link it in a comment

Reply:
fixed